sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -260,6 +262,13 @@ def build_list(state, arg):
     state.stack[-arg:] = [List[reduce(union, state.stack[-arg:], Union[()])]]
 
 
+def build_set(state, arg):
+  if arg == 0:
+    state.stack.append(List[Union[()]])
```

Comment:
Why are we appending a `List` instead of `Set` to the stack?

Reply:
Good catch. This should be list. I think this is missing in our tests because it's there's no empty set literal in Python, but I added some tests for set comprehension that exercise this.